README.md

```diff
@@ -13,6 +13,20 @@ cd research-computing-with-cpp
 ./build.sh
 ```
 
+You will need to have a bunch of stuff installed in order for the build to succeed:
+* Libraries:
+   * `brew install boost`
+   * `brew tap homebrew/science`
+     `brew install insighttoolkit`
```

Comment:
Also, you may want to mention this is for Mac. Linux users will know what to do for their distributions.
Windows users should be pointed to chocolatey (we should check whether insighttoolkit is there or not)